# Notes on talks for the Joint Statistical Meetings (JSM) 2016

* [JSM Online Program](https://www.amstat.org/meetings/jsm/2016/onlineprogram/index.cfm)
* A pdf of the [JSM Program can be found here](https://www.amstat.org/meetings/jsm/2016/pdfs/JSM2016-ProgramBook.pdf)
* Follow [#JSM2016 tweets here](https://twitter.com/search?q=%23jsm2016&src=typd)
* There is even a [JSM 2016 app](https://www.amstat.org/meetings/jsm/2016/jsmapp.cfm) that you can download
* [Links to slides from JSM 2016](https://github.com/kbroman/JSM2016slides)

##  My agenda 

These are the sessions I'm interested in attending. I know there a lot listed in the same time slots, but I'm hoping to catch a few talks in different ones. The numbers at the beginning list the session number.  If anyone is interested, my talk is on Tues 2-3:50pm in [Session 405 titled Statistical Challenges in the Analysis of Single-Cell RNA-Seq Data](https://www.amstat.org/meetings/jsm/2016/onlineprogram/ActivityDetails.cfm?SessionID=212483)

#### Sunday 

2-3:50pm

* 2 - Novel Statistical Methods in Imaging (Invited)
	* 2:05pm - From Statistical Visual Modeling and Computing to Communicative Learning ([Tianfu Wu](http://www.stat.ucla.edu/~tianfu.wu/), UCLA but soon NCSU)
		* "dark learning" = want to go beyond deep learning to understand the social meaning in images
		* Visual [Turning Test](https://en.wikipedia.org/wiki/Turing_test) in images - you can ask, answer and test story-like questions
		* Classical assumption for tracking is to use [Hidden Markov Models](https://en.wikipedia.org/wiki/Hidden_Markov_model) and [Particle Filtering](https://projecteuclid.org/euclid.ss/1280841735) (tries to integrates out previous state to predict current state). Big challenges: high-dimensional space and in-homogenous state spaces. 
	* 2:30pm - Nonparametric Spatio-Temporal Analysis of Neuroimaging Data ([Nathaniel Helwig](http://users.stat.umn.edu/~helwig/), U of Minnesota)
		* [Bilinear model](https://www.quora.com/What-are-some-examples-of-bilinear-models) - e.g. PCA, factor analysis, ICA. Across the rows is the temporal relationship. Across the columns is the spatial relationship. Here PCA "scores" are the temporal components and "loadings" are the spatial components. 
			* Major caveats (can rotate model without changing fit). **Rotational indeterminacy problem** = pick any orthogonal rotation matrix. we can define a new rotation. To solve in practice, people assume this bilinear model form and assume the columns (or rows) are independent. Tensor
		* Models for three-way data. Argues these are under utilized models (i.e. multi-mode, multi-linear)
			* The Covariation Chart by Raymond Cattell (1952) - first illustration of a three-way array
			* Tucker's three-way factor analysis model (1966) - assumes a cube of data is composed of a smaller cube
			* Parafac (parallel factors) - popular in chemistry. No rotational indeterminacy problem (yay!)
		* Tensor models with smoothness constraints. Can analyze variation across time, space and samples. Suggests using the parafac model, but uses a smoothed component function across the ith time point and jth spatial point. 
			* e.g. B-splines basis, smooth splines, second-order difference penalty. (All adds big computational costs due as does sample size)
		* [bigsplines R pkg](https://cran.r-project.org/web/packages/bigsplines/index.html) = Smoothing Splines for Large Samples
	* 2:55pm - Bayesian Feature Screening for Big Neuroimaging Data via Massively Parallel Computing ([Jian Kang](http://www-personal.umich.edu/~jiankang/), U of Michigan)
		* **structural imaging** (measures brain structures or shows contrasts between brain tissues e.g. MRI) vs **functional imaging** (measures neural activity and makes inference on the brain function)
		* Data: [Autism brain imaging data exchange (ABIDE)](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4162310/) - Goal is to develop a biomarker using neuro-imaging data to predict risk of disease. Biomarker used is the Voxel-wise fALFF value (captures local brain activity). Total of 1112 individuals (~500 cases, ~500 controls). Data from individual captured includes 175K voxels in 116 ROIs (regions of interest). 
		* Models: "Scalar-on-Image Regression". Discussed many variable selection methods and "ultra-high dimensional [variable screening](http://www.stat.ncsu.edu/people/bloomfield/courses/st430-514/slides/MandS-ch06-sec01-04.pdf)" methods. Author extends models to a bayesian framework for the Posterior Mean Screening.
	* 2:50pm - Joint Analysis of Brain Imaging Data and Genetics Data ([Wenxuan Zhong](http://www.stat.uga.edu/people/wenxuan-zhong), U of Georgia)
* 6 - Open Source Statistical Software for Data Science (Invited)
	* 2:05pm - Software Engineering for Data Science (Skipper Seabold, Civis' Analytics)
	* 2:25pm - The Python Data Science Stack ([Jake VanderPlas](http://staff.washington.edu/jakevdp/), U of Washington) [[Slides](https://speakerdeck.com/jakevdp/pythons-data-science-stack-jsm-2016)]
	* 2:45pm - Flexibility and Speed: Can We Have Both? ([Douglas Bates](https://www.stat.wisc.edu/~bates/), U of Wisconsin)
		* [lme4](http://r-forge.r-project.org/projects/lme4/) - Linear, generalized linear and nonlinear mixed models in R
	* 3:05pm - If You Can't Beat 'Em ([Dirk Eddelbuettel](http://dirk.eddelbuettel.com), Ketchum Trading/Debian & R Projects)
	* 3:25pm - Grammars and Structures for Computing with Data (Michael Kane, Yale University) [[Slides](http://slides.com/michaelkane/deck-17#/)]
* 10 - Statistical Methods in Integrative Genomics (Invited)
* 26 - Distinguishing Between Statistics Education for Undergraduate and Graduate Nonstatistics Major Students (Topic-Contributed)
* 33 - Efficient Methods for Structured Large Genomics Data (Contributed)

4-5:50pm

* 45 - Introductory Overview Lecture: Spatio-Temporal Data Analysis (Invited)
* 47 - Making the Most of R Tools (Invited)
	* 4:05pm- Thinking with Data Using R and RStudio: Powerful Idioms for Analysts ([Nicholas Horton](https://www.amherst.edu/people/facstaff/nhorton), Amherst College) [[Slides](https://github.com/Amherst-Statistics/JSM2016-thinkwithR)]
	* 4:35pm - Transform Your Work Flow and Deliverables with Shiny and R Markdown ([Garrett Grolemund](https://twitter.com/statgarrett), Rstudio)
		* [RStudio Cheatsheets](https://www.rstudio.com/resources/cheatsheets/) - RMarkdown, IDE, Shiny, Data Viz, Package Development, Data Wrangling, Base R, Advanced R
		* [htmlwidgets](http://www.htmlwidgets.org) - Use JavaScript data visualization libraries in R (i.e. plots). Embed widgets in RMarkdown documents and Shiny web applications.
* 51 - Media and Statistics (Invited)
	* 4:05pm - Causal Inferences from Observational Studies: Fracking, Earthquakes, and Oklahoma (Howard Wainer, NBME)
	* 4:25pm - It's Not What We Say, It's Not What They Hear, It's What They Say They Heard ([Barry Nussbaum](http://magazine.amstat.org/blog/2012/02/01/nussbaum/), EPA, [ASA President-Elect 2016](http://magazine.amstat.org/blog/2015/07/01/2015electionresults/))
	* 4:45pm - Bad Statistics, Bad Reporting, Bad Impact on Patients: The Story of the PACE Trial ([Julie Rehmeyer](http://julierehmeyer.com), Discover Magazine)
		* [PACE Trial](http://www.wolfson.qmul.ac.uk/current-projects/pace-trial/) = deeply flawed [trial on chronic fatigue syndrome (CFS) published in The Lancet in 2011](http://www.thelancet.com/journals/lancet/article/PIIS0140-6736(11)60096-2/abstract). Impacts public health recommendations world-wide. 
		* Results from trial states psychotherapy and exercise can help with CFS. Turns out it had flawed definition of what is an "improvement" (i.e. you could actually get worse and be defined as "better"). 
	* 5:05pm - Can Statisticians Enlist the Media to Successfully Change Policy? ([Don Berry](http://faculty.mdanderson.org/Donald_Berry/), MD Anderson) 
* 56 - Extraordinary Impact of Statistics (Invited) 
	* 4:05pm - A Short History of Statistical Ideas (David Siegmund, Stanford U)
	* 4:30pm - Cutting-Edge Research in Modern Statistical Sciences: Modern Tools and Impact in Data Science ([Heike Hofmann](http://hofmann.public.iastate.edu), ISU)
	* 4:55pm - Women in Statistics: Past, Present, and Future (Sally Morton, U of Pittsburgh)
	* 5:20pm - Disc ([Xiao-Li Meng](http://statistics.fas.harvard.edu/people/xiao-li-meng), Harvard)
* 61 - Modeling Multivariate Count Data: Multivariate Extensions and Generalizations of Standard Count Distributions (Topic-Contributed)
* 74 - Estimation and Learning in Graphical Models (Contributed)
	* 4:35pm - An Exposition on the Propriety of Restricted Boltzmann Machines ([Andee Kaplan](http://andeekaplan.com), ISU) [[Slides](http://andeekaplan.com/rbm/presentations/jsm2016/)]
* 78 - For the Love of the Game: Applicatsions of Statistics in Sports (Contributed)
	* 4:20pm - Estimating NCAA Football Coaches' Abilities: An Application of Item Response Theory (Brandon LeBeau, U of Iowa) [[Slides](http://educate-r.org/2016/07/31/jsm2016.html)]
* 81 - Clustering Methods (Contributed)

#### Monday 

8:30-10:20am

* 96 - Introductory Overview Lecture: Causal Inference (Invited)
* 106 - Applied Data Visualization in Industry and Journalism (Invited)
	* 8:35am - Linked Brushing in R (Hadley Wickham, Rice U)
		* [modelr](https://github.com/hadley/modelr) - helper functions for modeling data
	* 8:55am - Creating Data Visualization Tools at Facebook (Andreas Gros, Facebook)
	* 9:15pm - Cocktail Party Horror Stories About Data Vis for Clients (Lynn Cherny, Ghostweather R&D)
		* [Data Vis Consulting: Advice for Newbies](http://blogger.ghostweather.com/2013/11/data-vis-consulting-advice-for-newbies.html)
	* 9:35pm - Visualizing the News at Five ThirtyEight (Andrei Scheinkman, FiveThirtyEight.com)
	* 9:55pm - Teaching Data Visualization to 100k Data Scientists: Lessons from Evidence-Based Data Analysis ([Jeffrey Leek](http://jtleek.com), Johns Hopkins SPH)
* 111 - Recent Statistical Developments in Cancer Research (Topic-Contributed)
	* 8:35am - Normalization for Single-Cell RNA-Seq ([Christina Kendziorski](https://www.biostat.wisc.edu/~kendzior/), U of Wisconsin) 
		* scnorm = Performs quantile polynomial regression within groups of genes (genes are grouped via K-means with distance depending on gene-specific expression level). Not published yet. 
			* Models expression as a function of sequencing depth within each group of genes. Assessed FC bias in simulations
			* Idea: If you plot sequencing depth vs median expression (of non-zeros), the slope is different for different levels of expression (in groups of genes with high and low expression)
	* 8:55am - Statistical Issues in Single-Cell Analysis for Cancer Research ([Omar De La Cruz Cabrera](http://epbiwww.case.edu/index.php/people/faculty/129-delacruzcabrera), Case Western)
* 116 - What's Wrong with P-Value? (Topic-Contributed)
* 118 - Challenges in Metagenomic Data Analysis: Reproducibility and Interpretability of Inferences on Microbial Community Composition and Dynamics (Topic-Contributed)
* 125 - Statistical Methods for Functional Data (Contributed)
* 127 - R Tools for Statistical Computing (Contributed)
	* 9:50am - Broom: An R Package for Converting Statistical Modeling Objects Into Tidy Data Frames ([David Robinson](http://varianceexplained.org), Stack Overflow)

10:30am-12:20pm

* 141 - Some New Perspectives in Statistical Analysis with Incomplete Data (Invited)
* 142 -  The Extraordinary Impacts of Statistics in Genomics and Genetics (Invited)
	* 10:35am - Latent Variable Methods for the Analysis of Genomic Data ([John Storey](http://www.genomine.org), Princeton) [[Slides](http://genomine.org/talks/jsm_2016.pdf)]
		* [jackstraw](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4325543/) -  Identify genomic variables that are statistically significantly associated with any subset or linear combination of PCs
	* 11:00am - Overcoming Bias and Batch Effects in RNAseq Data ([Michael Love](http://mikelove.github.io), [Rafael A. Irizarry](http://rafalab.dfci.harvard.edu), DFCI)
		* [alpine](https://github.com/mikelove/alpine) - Modeling and correcting fragment sequence bias for RNA-seq
	* 11:25am - Testing High-Dimensional Differential Matrices, with Applications to Detecting Schizophrenia Genes ([Kathryn Roeder](http://www.stat.cmu.edu/~roeder/), Carnegie Mellon U)
	* 11:50am - Graph-Restricted Mixture Models ([Michael Newton](http://www.stat.wisc.edu/~newton/), U of Wisconsin)
* 159 - Biostatistical Literacy: How Best to Teach Medical and Public Health Professionals What They Need to Know About Statistics (Topic-Contributed)
* 169 - SPEED: Statistical Computing and Sports? (Contributed)

2-3:50pm

* 222 - Tricks and Treats in Classification and Regression Trees (Invited)
* 227 - Statistical Foundations of Data Privacy (Invited)
* 242 - Methods for Genetics and Genomics Data (Contributed)

4:45-6:15pm

* 272 - ASA President's Invited Address (Invited) 
	* [Joe Palca](http://www.npr.org/people/2101004/joe-palca), NPR - Science and News: A Marriage of Convenience

#### Tuesday 

8:30-10:20am

* 280 - Introductory Overview Lecture: Data Science
* 285 - New Advances in Statistical Genetics for Large-Scale Genomic Data (Invited)
* 286 - Women in Statistics: Past, Present, Future (Invited)
* 302 - Advanced Statistical Methods for High- Dimensional Microbiome Data Analysis (Topic-Contributed)
* 318 - Novel Approaches for Metagenomic, Phylogenetic, and Epigenetic Analysis (Contributed)
* 325 - Late-Breaking Session II: Data Journalism and Statistical Expertise: An Urgent Need for Writers, Bloggers, and Journalists to Be Statistically Savvy (Invited)

10:30am-12:20pm

* 327 - Statistics in Personalized Medicine (Invited)
* 332 - Doing More with Data in and Outside the Undergraduate Classroom (Invited)
* 339 - Big Data Challenges and Statistical Advances in Functional Genomics (Topic-Contributed)
* 334 - Novel Missing Data Imputation Methods (Topic-Contributed)
* 356 - Methods for Next-Generation Sequencing Data (Contributed)
* 357 - Advances in Statistical Genetics and Genomics (Contributed)

2-3:50pm

* 403 - New Methods for Detecting Sparse and Weak Effects in Genetic/Genomic Data (Invited)
* 405 - Statistical Challenges in the Analysis of Single-Cell RNA-Seq Data (Invited)
	* 2:05pm - [MAST](https://github.com/RGLab/MAST): A Novel Statistical Framework for Assessing Transcriptional Changes and Characterizing Heterogeneity in Single-Cell RNA-Seq Data ([Andrew McDavid](https://github.com/amcdavid), Fred Hutch/U Rochester)
		* Cellular detection rate may reflect the size of the cell (see [Padovan-Merhar et al. (2015)](http://www.ncbi.nlm.nih.gov/pubmed/25866248)) - McDavid is waiting experimental data to confirm this	* 2:30pm - [Yoav Gilad](http://giladlab.uchicago.edu) - [Batch effects and the effective design of single-cell gene expression studies](http://biorxiv.org/content/early/2016/07/08/062919)
		* Conversion efficiency from reads to molecules is affected by individual and technical C1 batch; goes against the idea of using UMIs as the absolute measurement of gene expression	* 2:55pm - Learning the 'Metadata' of the Cell: Inferring Cellular Phenotypes with Single-Cell Genomics ([Rahul Satija](http://www.satijalab.org), NY Genome Center)
		* [seurat](http://www.satijalab.org/seurat.html) - [Spatial reconstruction of single-cell gene expression data](http://www.nature.com/nbt/journal/v33/n5/full/nbt.3192.html)	* 3:20pm - [On the Widespread and Critical Impact of Systematic Bias and Batch Effects in Single-Cell RNA-Seq Data](http://biorxiv.org/content/early/2015/12/27/025528) ([Stephanie Hicks](http://www.stephaniehicks.com), DFCI/Harvard)
* 406 - Recent Advances in High-Dimensional Statistics and Computational Method (Invited)
* 407 - Interactive Visualizations and Web Applications for Analytics (Invited)
	* 2:35pm - Composable Linked Interactive Visualizations in R with Htmlwidgets and Shiny (Joseph Cheng, RStudio)
		* [crosstalk](https://github.com/rstudio/crosstalk) -  R pkg for interactive web graphs. Define in R, deploy in Javascript. 
* 434 - Statistical Modeling of RNA-Seq Data (Contributed)
	* 2:20pm - [TSCAN](https://www.bioconductor.org/packages/release/bioc/html/TSCAN.html): Pseudo-Time Reconstruction and Evaluation in Single-Cell RNA-Seq Analysis (Zhicheng Ji and [Hongkai Ji](http://www.biostat.jhsph.edu/~hji/), JHU)

In the afternoon

* 454 - Deming Lecture 
* President's address (8-9:30pm)

#### Wednesday 

8:30-10:20am

* 465 - Data Science for Health Policy: A Broad Tent (Invited)
* 471 - New Statistical Methods for the Analysis of High-Dimensional Biomarkers (Invited)
* 475 - Reproducibility in Statistics and Data Science (Invited) 
	* 8:35am - Reproducibility for All and Our Love/Hate Relationship with Spreadsheets ([Jenny Bryan](http://www.stat.ubc.ca/~jenny/), U of British Columbia) [[Slides](https://github.com/jennybc/2016-06_spreadsheets#readme)]
		* [googlesheets](https://cran.r-project.org/web/packages/googlesheets/vignettes/basic-usage.html)	* 8:55am - Steps Toward Reproducible Research ([Karl Broman](http://kbroman.org), U of Wisconsin) [[Slides](https://www.biostat.wisc.edu/~kbroman/presentations/repro_research_JSM2016_withnotes.pdf)]	* 9:15am - Enough with Trickle-Down Reproducibility: Scientists, Open  is Gate! Scientists, Tear Down  is Wall! ([Karthik Ram](http://karthik.io), UC Berkeley)	* 9:35am - Integrating Reproducibility into the Undergraduate Statistics Curriculum ([Mine Cetinkaya-Rundel](http://www2.stat.duke.edu/~mc301/), Duke) [[Slides](https://github.com/mine-cetinkaya-rundel/2016-08-03-reproducible-undergrad-stats)]	* 9:55am - Disc: [Yihui Xie](http://yihui.name/en/), RStudio
* 499 - Statistical Learning Approaches to Biological Inference Problems (Contributed)
* 501 - Analysis of Gene Expression, Genomics, and Next-Generation Sequencing Dat (Contributed)

10:30am-12:20pm

* 509 - Social Networks as the Unit of Observation (Invited)
* 511 - Statistical Methods for Analyzing Microbiome Data (Invited)
	* 10:35am - Bayesian Variable Selection Models for Microbiome Data Integration (Duncan Wadsworth, Rice/Microsoft)
	* 11:00am - Statistical Methods for Integrating the Phylogenetic Tree in Microbiome Data Analysis (Jun Chen, Mayo Clinic)
	* 11:25am - Kernel Penalized Regression Models for Microbiome Data (Timothy Randolph, Fred Hutch)	* 11:50am - Analysis of Composition of Microbiome with Structural Zeros (ANCOMSZ) (Shyamal Peddada, NIEHS)
* 513 - Recent Advances in Functional Data Analysis (Invited)
* 517 - Do Courts Appreciate the Power of Statistical Evidence? (Invited)
* 533 - Modeling Confounders via Smoothing and Regularization Methods? The Case of Age-Period Cohort and Beyond (Topic-Contributed)

2-3:50pm

* 579 - Challenges and Opportunities for Analysis of High-Dimensional and Big Data (Invited)
* 580 - Statistical and Computational Advances in Microbiome and Metagenomic Studies (Invited)
* 586 - Collaboration Among Academia, Industry, and Government, and the Role of ASA (Invited)
* 587 - Resampling Methods for High-Dimensional Inference (Invited)
* 593 - Batch Effects in Genomics Data (Topic-Contributed)
	* 3:05pm - Accounting for Sample Quality and Other Unwanted Effects in Single-Cell RNA-Seq Data ([Davide Risso](http://www.stat.berkeley.edu/~davide/Personal_Page/Home.html), UC Berkeley)

#### Thursday

8:30-10:20am

* 632 - Julia for Modern Statistical Computing (Invited)
* 634 - Analysis, Storage, and Privacy for Big Data (Invited)
* 649 - Clustering, Classification, and Dimension Reduction Techniques (Contributed)
* 655 - New Advances in Clustering Algorithms (Contributed)
* 656 - Hypothesis Testing for Correlation and Dependence (Contributed)

10:30am-12:20pm

* 678 - Strategies for Developing Undergraduate Data Science Programs (Invited)
* 686 - Statistics for Social Good (Topic-Contributed)
* 691 - Modern Biosurveillance at the Edge of Online Social Media, Social Networks, and Nontraditional Big Data (Topic-Contributed)


