# Notes on talks for the 2016 SCG Conference

Notes and slides for the [2016 Single Cell Genomics (SCG) Conference](https://coursesandconferences.wellcomegenomecampus.org/events/item.aspx?e=596) 
at the Wellcome Genome Campus, Hinxton, Cambridge, UK from Sept 14-16, 2016. Here is a pdf of the 
[SCG Program](http://conf.hinxton.wellcome.ac.uk/advancedcourses/SSG16draftprogrammeFinal.pdf). 

Pull requests welcome! or tweet me
[@stephaniehicks](https://twitter.com/stephaniehicks). 


## Wednesday, Sept 14 

### Keynote Lecture

- [Arnold Kriegstein](https://bms.ucsf.edu/directory/faculty/arnold-kriegstein-md-phd), Genomic insights into human cortical development, lissencephaly, and Zika microcephaly

### Session 1: Neuroscience & Tissue Development

**Chair**: [Rickard Sandberg](http://sandberg.cmb.ki.se) from [@karolinskainst](https://twitter.com/karolinskainst)

- [Sten Linnarsson](http://linnarssonlab.org), [@slinnarsson](https://twitter.com/slinnarsson), Molecular census of mouse brain cell types

- [Barbara Treutlein](http://www.treutleinlab.org), Reconstructing human organogenesis using single-cell RNA- seq

- [Maria Kasper](http://ki.se/en/people/markas), Plasticity and heterogeneity of skin cells in health and tissue repair

- [Naomi Habib](http://zlab.mit.edu/team.html), Single nucleus RNA-Seq reveals dynamics of adult neurogenesis

- [Gray Camp](http://www.eva.mpg.de/genetics/staff.html), Human cerebral organoids recapitulate gene expression programs of fetal neocortex development

- [Marta Rodriguez Orejuela](https://www.mdc-berlin.de/10179514/en/research/research_teams/systems_biology_of_gene_regulatory_elements/team), Characterization of adult neurogenic niches at single cell resolution



## Thursday, Sept 15

### Session 2: Chromatin Structure and Organization 

**Chair**: [Ido Amit](https://www.weizmann.ac.il/immunology/AmitLab/front) from [@WeizmannScience](https://twitter.com/WeizmannScience)

- [Amos Tanay](https://compgenomics.weizmann.ac.il/tanay/), Single cell dynamics of clonal memory

- [Will Greenleaf](http://greenleaf.stanford.edu/index.html), [@WJGreenleaf](https://twitter.com/wjgreenleaf), ATAC-ing regulatory variation in single cells

- [Stephen Clark](http://www.babraham.ac.uk/our-research/lymphocyte/geoffrey-butcher/members/65/stephen-clark), Chromatin accessibility, DNA methylation and gene expression from the same single-cell

- [Ana Pombo](https://pombolab.wordpress.com), [@apombo1](https://twitter.com/apombo1), Genome Architecture Mapping, new approach to map chromatin contacts

- [Peter Fraser](http://www.babraham.ac.uk/our-research/nuclear-dynamics/peter-fraser), [@Peter_Fraser1](https://twitter.com/peter_fraser1), Chromosome dynamics revealed by single cell HiC

- [Amanda Ackerman](http://www.chop.edu/doctors/ackermann-amanda#.V9F6JmX_RGI), Single-cell ATAC-seq identifies epigenetic differences in human pancreatic islet cell subtypes from normal and diabetic donors

- [Jan-Philipp Mallm](https://malone.bioquant.uni-heidelberg.de/people/mallm/index-mallm.html), Dissecting Deregulated Enhancer Activity in Primary Leukemia Cells



### Session 3: Immunology and Cancer

**Chair**: [John Marioni](http://www.ebi.ac.uk/research/marioni) from [@emblebi](https://twitter.com/emblebi) and [@CRUKresearch](https://twitter.com/crukresearch)

- [Sarah Teichmann](http://www.teichlab.org), Understanding Cellular Heterogeneity

- [Ido Amit](https://www.weizmann.ac.il/immunology/AmitLab/front), Immunology in the age of single cell genomics

- [Timm Schroeder](https://www.bsse.ethz.ch/department/people/detail-person.html?persid=193443), Long-term single cell quantification: New tools for old questions

- [Amir Giladi](http://www.weizmann.ac.il/lifesci/idcards/AmirGiladi0464.html), Transcriptional heterogeneity and lineage commitment in hematopoietic progenitors

- [Bart Deplancke](http://deplanckelab.epfl.ch), [@BartDeplancke](https://twitter.com/bartdeplancke), Single-cell RNA-seq-based identification and characterisation of somatic stem cells in adipose tissue & beyond

- [Matan Hofree](https://www.researchgate.net/profile/Matan_Hofree), Unbiased whole tissue analysis of the single cell transcriptional landscape of colon cancer

- [Fabian Theis](http://fabian.theis.name), Diffusion pseudotime identifies lineage choice and graded transitions in myeloid progenitors



## Friday, Sept 16

### Sesssion 4: Transcriptomics

**Chair**: [Sten Linnarsson](http://linnarssonlab.org), [@slinnarsson](https://twitter.com/slinnarsson)

- [Rickard Sandberg](http://sandberg.cmb.ki.se), Single-cell gene expression analyses of allelic transcription and regulation

- [Allon Klein](http://klein.hms.harvard.edu), Population balance reconstruction of differentiation hierarchies in developing and adult tissues by single cell droplet RNA-Seq

- [Arnau Sebé-Pedrós](https://compgenomics.weizmann.ac.il/tanay/?page_id=12), [@ArnauSebe](https://twitter.com/arnausebe), Early metazoan cell type evolution by single cell RNA-seq analysis

- [Omid Faridani](https://www.researchgate.net/profile/Omid_Faridani2), Sequencing Small-RNA transcriptome of individual cells

- [Alexander van Oudenaarden](http://www.hubrecht.eu/onderzoekers/van-oudenaarden-group/), Revealing novel cell types, cell-cell interactions, and cell lineages by single-cell sequencing

- [Stephanie Hicks](http://www.stephaniehicks.com), [@stephaniehicks](https://twitter.com/stephaniehicks), On the widespread and critical impact of systematic bias and batch effects in single-cell RNA-Seq data

- [Marc Wadsworth](https://www.researchgate.net/profile/Marc_Wadsworth), Seq-Well: A Portable Single-Cell RNA-Seq Platform for Low-Input Clinical Samples

- [Eshita Sharma](https://scholar.google.com/citations?user=xljyFDkAAAAJ&hl=en), Single cell preservation for RNAseq

- [Marc Unger](https://www.fluidigm.com/about/aboutfluidigm), Single-cell transcriptomics and functional analysis of single- cells

- [Manuel Garber](http://garberlab.umassmed.edu), Dissection of T1 Diabetes progression using Single cell RNA sequencing of a RAT model



### Sesssion 5: Imaging and Modeling

**Chair**: [Alexander van Oudenaarden](http://www.hubrecht.eu/onderzoekers/van-oudenaarden-group/) from [@_Hubrecht](https://twitter.com/_hubrecht?lang=en)

- [Long Cai](http://singlecell.caltech.edu/cailab/), In situ transcription profiling in tissues by seqFISH

- [Heather Lee](http://www.babraham.ac.uk/our-research/epigenetics/olivia-casanueva/members/198/heather-lee), Dynamic and heterogeneous DNA methylation in pluripotent cells

- [Steffen Rulands](http://www.rulands.net), [@srulands](https://twitter.com/srulands), Dynamic and heterogeneous DNA methylation in pluripotent cells

- [Jeffrey Moffitt](https://scholar.google.com/citations?user=U7eic7AAAAAJ&hl=en), High-throughput, spatially resolved, single-cell transcriptomics with MERFISH

- [Jan Philipp Junker](https://scholar.google.com/citations?user=0tt8A_4AAAAJ), Massively parallel clonal analysis using CRISPR/Cas9 induced genetic scars

- [John Marioni](http://www.ebi.ac.uk/research/marioni), Dissecting cell fate choice using single-cell genomics

- [David van Dijk](https://sciencedavid.wordpress.com), MAGIC: A Diffusion based data imputation method reveals progressions and gene-gene interactions in breast cancer cells undergoing EMT

- [Rom Shenhav](http://shalevlab.weizmann.ac.il/group-members/), Single-cell spatial reconstruction reveals global division of labor in the mammalian liver

- [Petra Schwalie](https://scholar.google.com/citations?user=EMSKH8cAAAAJ&hl=en), Accurate identification of somatic stem cells using single-cell RNA-sequencing




