# Notes on talks for the Bioconductor (BioC) 2017

* [BioC 2017 website](http://bioconductor.org/help/course-materials/2017/BioC2017/), held July 26-28 in Boston, MA
* Follow [#BioC2017 tweets here](https://twitter.com/search?q=%23bioc2017&src=typd)
* Notes: Birds-of-a-feather (BOF)

## Wednesday July 26 (Developer Day)

9:00-10:30am

* Welcome
* Lightning Talks I
* Community Activity

11:00am-12:00pm

* Birds-of-a-feather I / Workshops I

1:00-2:30pm 

* Lightning Talks II
* Birds-of-a-feather II / Workshops II

3:00-4:30pm 

* Lightning Talks III
* Workshops III

4:30-5:00pm

* Panel discussion: project directions and opportunities

## Thursday July 27

Morning Talks

* 8:30-9:15am: Christina Kendziorski (University of Wisconsin) ? Statistical methods for single-cell RNA-seq data
* 9:15-10:00am: Rahul Satija (New York Genome Center, NYU) ? Challenges and opportunities in analysis of single cell transcriptomics
* 10:30-11:15am: James Lindsay (DFCI) ? Knowledge Systems at Dana Farber
* 11:15-11:30am: Stephanie Hicks (DFCI) ? Estimating cell type composition in whole blood using differentially methylated regions
* 11:30-11:45am: Stephen Piccolo (Brigham Young University)
* 11:45am-12:00pm: Nils Gehlenborg (Harvard Medical School) ? Relaxation Techniques for the Upset Data Scientist

Lunch (BOF) 

* Infrastructure for efficient storage and processing of large-scale single-cell genomics data. Davide Risso (Division of Biostatistics and Epidemiology, Weill Cornell Medical College) et al. Yawkey 306.

1-2:45pm (Workshops)

* Introduction to R and Bioconductor. Lori Shepherd (Roswell Park Cancer Institute). Smith 308.
* A Journey of Discovery through the GenomicRanges Infrastructure. Michael Lawrence (Genentech). Yawkey 307.
* Microbiome Data Analysis. Levi Waldron (CUNY School of Public Health) et al. Yawkey 306.
* Design and evaluate guide RNAs for CRISPR-Cas9 genome-editing using CRISPRseek and GUIDEseq. Lihua Julie Zhu (University of Massachusetts Medical School). Smith 309.

3:15-5pm (Workshops)

* Differential Gene Expression analysis using R / Bioconductor. Radhika Khetani (Harvard T. H. Chan School of Public Health) et al. Yawkey 306.
* Interactive visualization and Data Analysis with epiviz web components. Jayaram Kancherla (University of Maryland, College Park) et al. Smith 308.
* Functional enrichment analysis of high-throughput omics data in Bioconductor. Ludwig Geistlinger, Levi Waldron (CUNY School of Public Health). Smith 309.
* Learn to leverage 70,000 human RNA-seq samples for your projects. Leonardo Collado Torres (Lieber Institute for Brain Development). Yawkey 307.


## Friday July 28

Morning Talks

* 8:30-9:15am: Elham Azizi (Memorial Sloan Kettering) ? Bayesian inference for single-cell clustering and imputing
* 9:15-10:00: Martin Aryee (Harvard) ? Analyzing 3-dimensional genome topology data
* 10:30-11:00: Davis McCarthy (EBI, Cambridge, UK) ? Using R and Bioconductor to explore genetic effects on single-cell gene expression
* 11:00-11:30: Raphael Gottardo (Fred Hutchinson Cancer Research Center) ? Statistical methods for the analysis of flow cytometry data
* 11:30-11:45: Keegan Korthauer (DFCI) ? An R package for detection and inference of differentially methylated regions from bisulfite sequencing
* 11:45-12:00: Jeff Gentry (Broad Institute) ? Cromwell & WDL: Bioinformatics pipelines at any scale

Lunch (BOF)

* Analyzing publicly available cancer genomics data from GTEx and TCGA with Bioconductor. Sonali Arora (Fred Hutchinson Cancer Research Center, Seattle). Yawkey 306.

1-2:45pm (Workshops)

* Understanding Bioconductor Annotation Packages. James MacDonald (University of Washington). Smith 308.
* Bioconductor workflow for single-cell RNA-seq data analysis: dimensionality reduction, clustering, and pseudotime ordering. Fanny Perraudeau (Division of Biostatistics, School of Public Health, University of California, Berkeley, CA, USA) et al. Yawkey 307.
* Integrative analysis workshop with TCGAbiolinks and ELMER. Tiago Chedraoui Silva (University of São Paulo / Cedars-Sinai) et al. Smith 309.
* CyTOF workflow: differential discovery in high-throughput high-dimensional cytometry datasets. Malgorzata Nowicka (University of Zurich) et al. Yawkey 306.

3:15pm-5pm (Workshops)

* Ensemble gene set enrichment analysis with EGSEA. Matt Ritchie (Walter and Eliza Hall Institute of Medical Research). Yawkey 306.
* Variant Annotation Workshop with FunciVAR, StateHub and MotifBreakR. Dennis J Hazelett (Cedars-Sinai Medical Center), Simon G Coetzee. Smith 308.
* Integrated analysis and visualization of ChIP-seq data using ChIPpeakAnno, GeneNetworkBuilder and TrackViewer. Jianhong Ou (University of Massachusetts Medical School) et al. Smith 309.
* Multi-omics data representation and analysis with MultiAssayExperiment. Marcel Ramos (CUNY School of Public Health) et al. Yawkey 307.

