# Notes on talks for the Bioconductor (BioC) 2017

* [BioC 2017 website](http://bioconductor.org/help/course-materials/2017/BioC2017/), held July 26-28 in Boston, MA
* Follow [#BioC2017 tweets here](https://twitter.com/search?q=%23bioc2017&src=typd)
* Notes: Birds-of-a-feather (BOF)

## Wednesday July 26 (Developer Day)

9:00-10:30am

* Welcome
	* [https://courses.bioconductor.org](https://courses.bioconductor.org) to load 
* Community Activity
* Spotlight talk from Aaron Lun: Using beachmat to explore 10x Genomics' 1M Cell Data Set
	* [beachmat](http://www.biorxiv.org/content/early/2017/07/24/167445) 
		* a common C++ API for accessing single-cell genomics data from a variety of R matrix types (base matrix, `dgeMatrix`, `RleMatrix`, `HDF5Matrix`, `dspMatrix`, etc)
		* If you have C++ code in your BioC package, you can now access data / write data to and from columns. 
		* beachmat is built on top of [Rcpp](https://cran.r-project.org/web/packages/Rcpp/index.html). Takes in aany R matrix to convert to an Robject. 
	* [TENxGenomics](https://github.com/mtmorgan/TENxGenomics) - to read `TENxMatrix` file into R. 
		* Analysis: Convert `TENxMatrix` into `HDF5Matrix`. Used [scran]() with C++ functions based on beachmat to analyze data (using desktop with 8GB ram, Core i5 processor). For cell cycle assignment, `cyclone` ran in 24 hours. For normalization by deconvolution, ran in 3 hours. Randomly sampled 10k cells and did PCA. projected all other cells on first two dimensions. 
		
* Lightning Talks I
	* Lucas Schiffer
		* Discussed new markdown to write [support site](https://support.bioconductor.org) posts on BioC
	* Peter Hickey, [slides](https://bioc2017.updog.co/DelayedMatrixStats_BioC_2017.html#1)
		* Discussed `DelayedMatrixStats` to support the `matrixStats` API for `DelayedMatrix` objects 
	* Nitesh Turaga
		* Overview of Bioconductor's GIT transition
	* Kevin Rue-Albrecht, [slides](https://www.dropbox.com/s/8nudiozu4gahlcv/Ligthning_TVTB_v2.pptx?dl=0)
		* Overview of the VCF Tool Box (TVTB): an effort to summarize and visualize variants. Read in VCF files, calculate variant frequency, apply VCF filter rules. 
	* Robert Castelo, [slides](https://www.dropbox.com/s/qoq6hsxk1xe5kf2/bioc17.pdf?dl=0)
		* Overview of GenomicScores: efficient storage and retrieval of genome-wide position-specific scores

11:00am-12:00pm

* Birds-of-a-feather I / Workshops I
	* Michael Lawrence
		* Discussion on interfaces for programming with data: making Bioconductor software more analyst-friendly	

1:00-2:30pm 

* Lightning Talks II
	* Shweta Gopaulakrishnan, Samuela Pollock -- Semantically rich containers for cloud scale genomics (performance considerations)
	* BJ Stubbs, Vince Carey -- Multiomic regulatory network construction with txRegQuery
	* Carles Hernandez-Ferrer -- Extending Bioconductor To Exposome Data Analysis
	* John Lawson -- MIRA: an R Package for Inferring Regulatory Activity from DNA Methylation Data
	* Lucas Schiffer -- curatedMetagenomicData: Human Microbiome Data Lightening Fast

* Birds-of-a-feather II / Workshops II

3:00-4:30pm 

* Lightning Talks III
* Workshops III
	* Herve Pages: Working with large arrays -- the DelayedArray package

4:30-5:00pm

* Panel discussion: project directions and opportunities

## Thursday July 27

Morning Talks

* 8:30-9:15am: Christina Kendziorski (University of Wisconsin). Statistical methods for single-cell RNA-seq data. 
	* Discussed [Oscope](http://bioconductor.org/packages/release/bioc/html/Oscope.html) to find oscillating genes, [SCnorm](https://github.com/rhondabacher/SCnorm) for normalization of only the non-zero genes, and [scDD](https://bioconductor.org/packages/release/bioc/html/scDD.html) to identify differential distributions. 
* 9:15-10:00am: Rahul Satija (New York Genome Center, NYU). Challenges and opportunities in analysis of single cell transcriptomics. 
	* Discussed [Seurat 2.0](http://satijalab.org/seurat/). 
		1. Discover gene-gene correlations that are consderved between conditions (or two+ datasets). 
			* Canonical correlation analyses. Projects two datasets into a low dimensional space that tries to find a common correlation. Then use non-linear method to shift CC vectors (non-linear method is needed b/c of potential shifts in cell types proportions in two conditions). Specifally use Dynamic Time Warping to align two ordered vectors that are out a phase shift. 
			* Negative control: cells from two different cell types do not align. 
		2. remove dataset-specific cells (discard cells that cannot be effectively mapped into a shared space. 
		3. normalize for changes in data set composition and scale. e.g. differences in feature scale (UMI vs FPKM, etc). Be robust to changes in population density. 
			* Data: MARS-Seq (n=2700, 1000 genes/cell) and Smart-Seq2 (n=784, 6000 genes/cell). 
			* Data: Aligned cells from two species to identify shared cell types in the pancreas between human and mouse. 
		4. Compare across conditions. 
* 10:30-11:15am: James Lindsay (DFCI). Knowledge Systems at Dana-Farber
* 11:15-11:30am: Stephanie Hicks (DFCI). Estimating cell type composition in whole blood using differentially methylated regions
* 11:30-11:45am: Stephen Piccolo (Brigham Young University)
* 11:45am-12:00pm: Nils Gehlenborg (Harvard Medical School). Relaxation Techniques for the Upset Data Scientist

Lunch (BOF) 

* Infrastructure for efficient storage and processing of large-scale single-cell genomics data. Davide Risso (Division of Biostatistics and Epidemiology, Weill Cornell Medical College) et al. 

1-2:45pm (Workshops)

* Introduction to R and Bioconductor. Lori Shepherd (Roswell Park Cancer Institute).
* A Journey of Discovery through the GenomicRanges Infrastructure. Michael Lawrence (Genentech). 
* Microbiome Data Analysis. Levi Waldron (CUNY School of Public Health) et al. 
* Design and evaluate guide RNAs for CRISPR-Cas9 genome-editing using CRISPRseek and GUIDEseq. Lihua Julie Zhu (University of Massachusetts Medical School). 

3:15-5pm (Workshops)

* Differential Gene Expression analysis using R / Bioconductor. Radhika Khetani (Harvard T. H. Chan School of Public Health) et al. 
* Interactive visualization and Data Analysis with epiviz web components. Jayaram Kancherla (University of Maryland, College Park) et al. 
* Functional enrichment analysis of high-throughput omics data in Bioconductor. Ludwig Geistlinger, Levi Waldron (CUNY School of Public Health). 
* Learn to leverage 70,000 human RNA-seq samples for your projects. Leonardo Collado Torres (Lieber Institute for Brain Development).
	* [recount2 workshop notes](http://research.libd.org/recountWorkshop/). 


## Friday July 28

Morning Talks

* 8:30-9:15am: Elham Azizi (Memorial Sloan Kettering). Bayesian inference for single-cell clustering and imputing
	* [Biscuit](https://genomicscomputbiol.org/ojs/index.php/GCB/article/view/46) - iteratively normalizes and clusters cells. 
* 9:15-10:00: Martin Aryee (Harvard). Analyzing 3-dimensional genome topology data
	* [hichipper](https://github.com/aryeelab/hichipper) - A preprocessing and QC pipeline for [HiChIP](http://www.nature.com/nmeth/journal/v13/n11/full/nmeth.3999.html) data
* 10:30-11:00: Davis McCarthy (EBI, Cambridge, UK). Using R and Bioconductor to explore genetic effects on single-cell gene expression
	* Use donor pooling to minimize batch effects and increase numbers of cells per cell lines. Sample 100s cells from mixed populations of cells. Can successfully recover DonorID after mixing cells using average allelic correlations. Call variants at sites in 1000 highly variable genes in bulk iPCs. Tested on Smart-Seq2 and 10x data. 
* 11:00-11:30: Raphael Gottardo (Fred Hutchinson Cancer Research Center). Statistical methods for the analysis of flow cytometry data
	* [COMPASS](http://www.nature.com/nbt/journal/v33/n6/full/nbt.3187.html) - Bayesian framework to model all observed cell subsets and select those most likely to have antigen-specific responses.
* 11:30-11:45: Keegan Korthauer (DFCI). An R package for detection and inference of differentially methylated regions from bisulfite sequencing
	* [dmrseq](https://github.com/kdkorthauer/dmrseq) - prioritizes DMRs from BS DNAm experiments. Computes region summary statistics that account for known covariates
* 11:45-12:00: Jeff Gentry (Broad Institute). Cromwell & WDL: Bioinformatics pipelines at any scale

Lunch (BOF)

* Analyzing publicly available cancer genomics data from GTEx and TCGA with Bioconductor. Sonali Arora (Fred Hutchinson Cancer Research Center, Seattle). 

1-2:45pm (Workshops)

* Understanding Bioconductor Annotation Packages. James MacDonald (University of Washington). 
* Bioconductor workflow for single-cell RNA-seq data analysis: dimensionality reduction, clustering, and pseudotime ordering. Fanny Perraudeau (Division of Biostatistics, School of Public Health, University of California, Berkeley, CA, USA) et al. 
	* [workshop notes](https://f1000research.com/articles/6-1158/v1): Discussed ZINB-WaVE, clusterExperiment and slingshot. 
* Integrative analysis workshop with TCGAbiolinks and ELMER. Tiago Chedraoui Silva (University of São Paulo / Cedars-Sinai) et al. 
* CyTOF workflow: differential discovery in high-throughput high-dimensional cytometry datasets. Malgorzata Nowicka (University of Zurich) et al.

3:15pm-5pm (Workshops)

* Ensemble gene set enrichment analysis with EGSEA. Matt Ritchie (Walter and Eliza Hall Institute of Medical Research). 
* Variant Annotation Workshop with FunciVAR, StateHub and MotifBreakR. Dennis J Hazelett (Cedars-Sinai Medical Center), Simon G Coetzee. 
* Integrated analysis and visualization of ChIP-seq data using ChIPpeakAnno, GeneNetworkBuilder and TrackViewer. Jianhong Ou (University of Massachusetts Medical School) et al. 
* Multi-omics data representation and analysis with MultiAssayExperiment. Marcel Ramos (CUNY School of Public Health) et al. 

